applications_tests/gls_sharp_navier_stokes_3d/flow_around_cuthollowsphere.output

```diff
@@ -13,8 +13,8 @@ Steady iteration:        1/2
 +------------------------------------------+
 |  Force  summary particles                |
 +------------------------------------------+
-particle_ID  time    T_x      T_y       T_z       f_x        f_y        f_z      f_xv      f_yv      f_zv       f_xp       f_yp      f_zp    
-          0 1.0000 0.678984 3.112272 63.626985 198.021894 -39.624418 -0.610866 26.339639 -4.238040 -0.335928 171.682254 -35.386378 -0.274938 
+particle_ID  time    T_x      T_y       T_z        f_x        f_y      f_z      f_xv      f_yv     f_zv      f_xp      f_yp      f_zp
+          0 1.0000 2.871845 9.793226 117.105376 321.947926 11.919979 4.093938 29.607130 0.480646 0.013561 292.340796 11.439332 4.080377
```

Comment:
Why have the rsults changed so much here???

Reply:
There is a lot of overconstrained cells.